dataactvalidator/validation_handlers/validationManager.py

```diff
@@ -221,8 +223,9 @@ def runValidation(self, jobId, interfaces):
                                 self.reportHeaders) as writer:
                 while not reader.isFinished:
                     rowNumber += 1
-                    #if (rowNumber % 1000) == 0:
-                    #    print("Validating row " + str(rowNumber))
+                    if (rowNumber % 100) == 0:
+                        CloudLogger.logError("VALIDATOR_INFO: ","JobId: "+str(jobId)+" validating/copying row " + str(rowNumber),"")
```

Comment:
We're moving validation out of this part, a better message might be "loading row"